sde_sampler/losses/oc.py

```diff
@@ -369,3 +373,124 @@ def eval(
         return BaseOCLoss.compute_results(
             rnd, compute_weights=compute_weights, ts=ts, samples=samples, xs=xs
         )
+
+
+class ExponentialIntegratorSDELoss(BaseOCLoss):
+    def __init__(self, *args, **kwargs):
+        # Only loss.method = KL implemented
+        super().__init__(*args, **kwargs)
+        self.alpha = kwargs.get("alpha", 1)
```

Comment:
This looks prone to errors. Can we add alpha and sigma to the signature in `__init__` and not retrieve it from `kwargs`?